change/@fluentui-react-charting-c1fdbc01-292c-4f2d-b96e-1e417b7c89da.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33532&commitId=ef1d1a47f03c4c106ceca381483f99da8ac9a7d5&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Rendering callout attached to a rectangle.chromium.png|1493| Changed|
|Callout.Left bottom edge.chromium.png|3022| Changed|
|Callout.Left top edge.chromium.png|1949| Changed|
|Callout.Right center.chromium.png|2396| Changed|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic.chromium.png|376| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>